driver-sync/src/main/com/mongodb/client/MongoChangeStreamCursor.java

```diff
@@ -24,17 +25,18 @@
  * <p>
  * An application should ensure that a cursor is closed in all circumstances, e.g. using a try-with-resources statement:
  * </p>
- * <blockquote><pre>
- * try (MongoChangeStreamCursor&lt;Document&gt; cursor = collection.find().cursor()) {
+ * <pre>{@code
+ * try (MongoChangeStreamCursor<ChangeStreamDocument<Document>> cursor = collection.watch().cursor()) {
  *     while (cursor.hasNext()) {
  *         System.out.println(cursor.next());
  *     }
  * }
- * </pre></blockquote>
+ * }</pre>
  *
  * @since 3.11
  * @param <TResult> The type of documents the cursor contains
  */
+@NotThreadSafe
```

Comment:
`MongoCursor` (the super-interface) has already been annotated with `@NotThreadSafe`. Added the annotation here to avoid user confusion. I can see how absence of the annotation can be argued for. Accordingly, I will undo this change without arguing in the event of being asked to do so.